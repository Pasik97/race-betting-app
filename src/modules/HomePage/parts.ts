import styled from 'styled-components';

export const WelcomeText = styled.p`
   margin: 0;
   font-size: 18px;
   line-height: 21px;
   text-align: center;
`;
