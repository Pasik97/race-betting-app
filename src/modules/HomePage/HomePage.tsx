import React from 'react';
import * as P from './parts';

const HomePage: React.FC = () => (
   <P.WelcomeText>
      Welcome in races betting application. Use navigation to see current races.
   </P.WelcomeText>
)

export default HomePage;
