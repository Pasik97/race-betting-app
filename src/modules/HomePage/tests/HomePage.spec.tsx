import { create } from "react-test-renderer";
import HomePage from "../HomePage";

describe('<HomePage />', () => {
   it('should render welcome text', () => {
      const component = create(<HomePage />);

      expect(component.toJSON()).toMatchSnapshot();
   });
});
