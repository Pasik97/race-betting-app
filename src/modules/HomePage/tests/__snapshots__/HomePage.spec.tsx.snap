// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render welcome text 1`] = `
<p
  className="sc-bdvvtL iYyzJq"
>
  Welcome in races betting application. Use navigation to see current races.
</p>
`;
